core/base/persistenceDiagramClustering/PersistenceDiagramClustering.h

```diff
@@ -395,15 +330,7 @@ namespace ttk {
       }
     }
 
-    {
-      std::stringstream msg;
-      msg << "[PersistenceDiagramClustering] Processed in "
-          << tm.getElapsedTime() << " s. (" << threadNumber_ << " thread(s))."
-          << std::endl;
-      dMsg(std::cout, msg.str(), infoMsg);
-    }
+    { printMsg("", 1, tm.getElapsedTime(), threadNumber_); }
```

Comment:
see https://github.com/topology-tool-kit/ttk/blob/dev/core/base/helloWorld/HelloWorld.h (l. 123) for an example of time logging